### PR TITLE
Release v1.5.2

### PR DESCRIPTION
**Pull request by thobrla**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 
+## v1.5.2
+
+* Various test fixes (#305, #308, #311, #312, #313, #314)
+* Add access token refresh error class that includes HTTP status (#310)
+* Python3 compatbility fixes (#316, #318, #329)
```
Comment (dhermes) [inline CHANGELOG.md]: s/compatbility/compatibility

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 
+## v1.5.2
+
+* Various test fixes (#305, #308, #311, #312, #313, #314)
```
Comment (dhermes) [inline CHANGELOG.md]: Drop this from the changelog (not necessary for users to see)

File: CHANGELOG.md (hunk 1 above)
Comment (thobrla) [inline CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (thobrla) [inline CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## v1.5.2
+
+* Add access token refresh error class that includes HTTP status (#310)
+* Python3 compatibility fixes (#316, #318, #329)
```
Comment (dhermes) [inline CHANGELOG.md]: 329 can be left out. #316 and #318 should be highlighted as Python3 compat for the django stuff

Comment (thobrla) [inline CHANGELOG.md]: Done.

**Comment by dhermes** [comment]
For my reference:

```
$ git log -1 --pretty=%H  # What is HEAD at time of doing "log"
42279caa52d3222dec460f923ad0c0e94b04eacb
$ git log v1.5.1..HEAD \
> | grep 'Merge pull request ' \
> | awk -F ' ' '{print $4}' \
> | sort
```

yields

#305
#306
#308
#310
#311
#312
#313
#314
#316
#318
#322
#324
#328
#329
#330
#336
#340

**Comment by dhermes** [comment]
LGTM

**Comment by dhermes** [comment]
@thobrla https://github.com/google/oauth2client/releases/tag/v1.5.2 created and waiting on

https://travis-ci.org/google/oauth2client/builds/91903047

to push to PyPI

**Pull request merged**